spotinst_sdk2/models/ocean/azure/__init__.py

```diff
@@ -0,0 +1,589 @@
+        self.max_memory_giB = max_memory_giB
+        self.max_vCpu = max_vCpu
+        self.min_memory_giB = min_memory_giB
+        self.min_vCpu = min_vCpu
+        self.min_gpu = min_gpu
+        self.max_gpu = max_gpu
+        self.min_data=min_data
+        self.min_nICs=min_nICs
+        self.series = series
+        self.vm_types = vm_types
+
+class VmSizes:
+    '''
+    # Arguments
+    filters: Filters
+    '''
+    def __init__(
+            self,
+            filters: Filters = none):
+        self.filters = filters         
+
+class VirtualNodeGroupTemplate:
+    '''
+    # Arguments
+    name: str
+    ocean_id: str 
+    auto_scale: AutoScale
+    availabilty_zones: List[str]
+    labels: Labels
+    node_count_limits: NodeCountLimits
+    node_pool_properties: NodePoolProperties
+    strategy: Strategy
+    tags: Tags
+    taints: Taints
+    vm_sizes: VmSizes
+    '''
+
+    def __init__(
+            self,
+            name: str = none,
+            ocean_id: str = none,
+            auto_scale: AutoScale = none,
+            availabilty_zones: List[str] = none,
+            labels: Labels = none,
+            node_count_limits: NodeCountLimits = none,
+            node_pool_properties: NodePoolProperties = none,
+            strategy: Strategy = none,
+            tags: Tags = none,
+            taints: Taints = none,
+            vm_sizes: VmSizes = none):
+        self.name=name
+        self.ocean_id=ocean_id
+        self.auto_scale=auto_scale
+        self.availability_zones=availabilty_zones
+        self.labels=labels
+        self.node_count_limits=node_count_limits
+        self.node_pool_properties=node_pool_properties
+        self.strategy=strategy
+        self.tags=tags
+        self.taints=taints
+        self.vm_sizes=vm_sizes              
+       
+class Ocean:
+    '''
+    # Arguments 
+    aks: OceanAKS
+    auto_scaler: AutoScaler
+    controller_cluster_id: str
+    health: Health
+    name: str
+    scheduling: Scheduling
+    virtual_node_group_template: VirtualNodeGroupTemplate
+    '''
+
+    def __init__(
+            self,
+            aks: OceanAKS = none,
+            auto_scaler: AutoScaler = none,
+            controller_cluster_id: str = none,
+            health: Health = none,
+            name: str = none,
+            scheduling: Scheduling = none,
+            virtual_node_group_template: VirtualNodeGroupTemplate = none):
+        self.aks=aks
+        self.auto_scaler=auto_scaler
+        self.controller_cluster_id=controller_cluster_id
+        self.health=health
+        self.name=name
+        self.scheduling=scheduling
+        self.virtual_node_group_template=virtual_node_group_template
+
+class LaunchNewNodes:
+    '''
+    # Arguments
+    adjustment: int
+    applicable_vm_sizes: List[str]
+    availability_zones: List[str]
+    min_cores_per_node: int
+    min_memory_giB_per_node: float
+    ocean_id: str
+    preferred_lifecycle: str
```

Comment:
Add enum class
<html>
<body>
<!--StartFragment-->

preferredLifecycle | stringEnum: "Spot" "Regular"The preferred life cycle to launch the node. If not specified, the life cycle is chosen accoring to the scaled VNG.
-- | --


<!--EndFragment-->
</body>
</html>